modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -16,16 +16,16 @@ The adapters, APIs, and bindings are then upgraded, and finally requests to prot
 
 Before you upgrade, ensure that your environment uses:
 
-* Anypoint Service Mesh v1.0.x
+* Anypoint Service Mesh v1.1.x
 * The most recent version of the Anypoint Service Mesh command line interface (CLI)
 +
-The CLI version must be v1.1.x and the Anypoint Service Mesh version must be v1.0.x.
-* Istio 1.6.0 or later for both control plane and data plane
+The CLI version must be v1.2.x and the Anypoint Service Mesh version must be v1.1.x.
+* Istio 1.7.x or 1.8.x or later for both control plane and data plane
```

Comment:
Replace with
* Anypoint Service Mesh v1.1.x
* CLI v1.2.x 
* Istio 1.7.x or 1.8.x or later for both control plane and data plane
...but how can the CLI version be 1.2.x already, before the upgrade?

Reply:
The CLI is standalone of the product version, to be able to upgrade to Service Mesh 1.2.x you need to use a CLI 1.2.x in order to do that

Reply:
Thanks for the clarification @dosten.